app/models/miq_schedule_worker/jobs.rb

```diff
@@ -121,6 +121,14 @@ def miq_report_result_purge_timer
     queue_work(:class_name => "MiqReportResult", :method_name => "purge_timer", :zone => nil)
   end
 
+  def archived_entities_purge_timer
```

Comment:
@durandom something to add to the pluggable backlog ... how can we allow providers (and separate provider repos) to engage with different timers?